python/cudf/cudf/core/column/column.py

```diff
@@ -357,14 +357,22 @@ def to_arrow(self) -> pa.Array:
                 pa.null(), len(self), [pa.py_buffer((b""))]
             )
 
-        return libcudf.interop.to_arrow(
+        result = libcudf.interop.to_arrow(
             libcudf.table.Table(
                 cudf.core.column_accessor.ColumnAccessor({"None": self})
             ),
             [["None"]],
             keep_index=False,
         )["None"].chunk(0)
 
+        if isinstance(self.dtype, cudf.Decimal64Dtype):
+            result = result.cast(
+                pa.decimal128(
+                    scale=result.type.scale, precision=self.dtype.precision
+                )
+            )
```

Comment:
Is this cast zero copy? Should we be using `result.view(...)` instead?

Reply:
Amazing! Doing so also lets us get rid of the xfail below.